### PR TITLE
Use more actionable error messages for hardware encoders

### PR DESCRIPTION
**Pull request by notr1ch**
### Description
Updates error messages when hardware encoders fail to be more actionable. Also some minor tone / grammar fixes.

### Motivation and Context
Telling a user to "Check your driver is up to date" is too open to interpretation, many users will use Windows Update or Device Manager to check for updates which doesn't help. Instead, let's explicitly tell them what is most likely to fix the problem.

### How Has This Been Tested?
Simulated an error and verified the links show as expected.

### Types of changes

- Tweak (non-breaking change to improve existing functionality)

### Checklist:

- [x] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

### CODE REVIEW

**Comment by WizardCM** [comment]
We do expose NVENC on Linux these days, and so the driver link is irrelevant to them. This might require a separate translation key.

**Comment by notr1ch** [comment]
Linux drivers are also available at that link.

**Comment by derrod** [comment]
They are but Nvidia drivers on Linux are often handled by the distribution itself and using Nvidia's Linux installer can be a bit weird. So our Linux people suggested that it should not be shown for Linux users so they instead check their OS's respective repositories for updates.

**Comment by notr1ch** [comment]
Do we have an appropriate page to send them to?

**Comment by norihiro** [comment]
Would you please start the commit title from a verb?

File: plugins/obs-ffmpeg/data/locale/en-US.ini [hunk 1]
```diff
@@ -90,17 +90,17 @@ Encoder.Error="Failed to open %1: %2"
...
 NVENC.BadGPUIndex="You have selected GPU %1 in your output encoder settings. Set this back to 0 and try again."
-NVENC.OutdatedDriver="Your current video card driver does not support this NVENC version, please update your drivers."
-NVENC.UnsupportedDevice="NVENC Error: Unsupported device. Check your video card supports NVENC and that the drivers are up to date."
-NVENC.TooManySessions="NVENC Error: Too many concurrent sessions. Try closing other recording software which might be using NVENC such as NVIDIA Shadowplay or Windows 10 Game DVR."
-NVENC.CheckDrivers="Please check your video drivers are up to date."
+NVENC.OutdatedDriver="The installed NVIDIA driver does not support this NVENC version, try <a href=\"https://obsproject.com/go/nvidia-drivers\">updating the driver</a>."
+NVENC.UnsupportedDevice="NVENC Error: Unsupported device. Check your video card <a href=\"https://obsproject.com/go/nvenc-matrix\">supports NVENC</a> and try <a href=\"https://obsproject.com/go/nvidia-drivers\">updating the driver</a>."
```
Comment (RytoEX) [inline plugins/obs-ffmpeg/data/locale/en-US.ini:101]:
```suggestion
NVENC.UnsupportedDevice="NVENC Error: Unsupported device. Check that your video card <a href=\"https://obsproject.com/go/nvenc-matrix\">supports NVENC</a> and try <a href=\"https://obsproject.com/go/nvidia-drivers\">updating the driver</a>."
```

**Review by RytoEX — CHANGES_REQUESTED** [review]
Wording nit, but otherwise I think this is fine.

**Review by RytoEX — APPROVED** [review]
Force pushed changes. Looks good to me.

**Pull request merged**